decidim-admin/app/controllers/decidim/admin/participatory_process_attachments_controller.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+require_dependency "decidim/admin/application_controller"
+
+module Decidim
+  module Admin
+    # Controller that allows managing all the attachments for a participatory
+    # process.
+    #
+    class ParticipatoryProcessAttachmentsController < ApplicationController
+      include Concerns::ParticipatoryProcessAdmin
+
+      def index
+        authorize! :read, Decidim::ParticipatoryProcessAttachment
+      end
+
+      def new
+        authorize! :create, Decidim::ParticipatoryProcessAttachment
+        @form = ParticipatoryProcessAttachmentForm.new
+      end
+
+      def create
+        authorize! :create, Decidim::ParticipatoryProcessAttachment
+        @form = ParticipatoryProcessAttachmentForm.from_params(params)
+
+        CreateParticipatoryProcessAttachment.call(@form, participatory_process) do
+          on(:ok) do
+            flash[:notice] = I18n.t("participatory_process_attachments.create.success", scope: "decidim.admin")
+            redirect_to participatory_process_attachments_path(participatory_process)
+          end
+
+          on(:invalid) do
+            flash.now[:alert] = I18n.t("participatory_process_attachments.create.error", scope: "decidim.admin")
+            render :new
+          end
+        end
+      end
+
+      def edit
+        @participatory_process_attachment = collection.find(params[:id])
+        authorize! :update, @participatory_process_attachment
+        @form = ParticipatoryProcessAttachmentForm.from_model(@participatory_process_attachment)
```

Comment:
@oriolgual the `FormFactory` has a `from_model` method:

https://github.com/AjuntamentdeBarcelona/decidim/blob/186ab62a74218e096215b1ac5e8304d69c63ca9a/decidim-core/app/controllers/concerns/decidim/form_factory.rb#L52